pyproject.toml

```diff
@@ -59,7 +60,7 @@ filesystem = "fontra.filesystem.projectmanager:FileSystemProjectManagerFactory"
 
 [project.entry-points."fontra.webcontent"]
 filesystem = "fontra.filesystem"
-
+localplugins = "fontra.localplugins"
```

Comment:
There were two blank lines after this before: please keep the formatting the same.